zdoom.vcproj

```diff
@@ -6704,6 +6704,18 @@
 				>
 			</File>
 		</Filter>
+		<Filter
+			Name="Portals"
+			>
+			<File
+				RelativePath=".\src\portal.cpp"

```

Comment:
so this implies that there's also a new 'portal.cpp' file? There no new file in this pull request. Also, if this is needed, you need to update src/CMakeLists.txt as well.